libs/sdk-ui-kit/src/DocumentHeader/HeaderIcon.tsx

```diff
@@ -0,0 +1,53 @@
+// (C) 2007-2020 GoodData Corporation
+import { Component } from "react";
+import isEqual from "lodash/isEqual";
+import { removeFromDom } from "../utils/domUtilities";
+
+interface IHeaderIconProps {
+    rel: "shortcut icon" | "apple-touch-icon";
+    type: "image/x-icon" | "image/png";
+    url?: string;
+}
+
+export default class HeaderIcon extends Component<IHeaderIconProps> {
+    static defaultProps = {
+        url: "",
+    };
+
+    favicon: HTMLLinkElement;
+
+    componentDidMount(): void {
+        this.createFavicon();
+    }
+
+    shouldComponentUpdate(nextProps: IHeaderIconProps): boolean {
+        return !isEqual(this.props, nextProps);
+    }
+
+    componentDidUpdate(): void {
+        this.removeFavicon();
+        this.createFavicon();
+    }
+
+    componentWillUnmount(): void {
+        this.removeFavicon();
+    }
+
+    createFavicon(): void {
+        if (this.props.url) {
+            this.favicon = document.createElement("link");
```

Comment:
You could use [`react-helmet`](https://www.npmjs.com/package/react-helmet) to avoid this direct DOM manipulation. It also provides functionality you get from `react-document-title`. Can you please try using the `react-helmet`?

Reply:
If you don't have the time, please create a follow-up ticket in the FET-512 epic.

Reply:
thank for understanding, I have created the ticket here: https://jira.intgdc.com/browse/FET-591

Reply:
No problem, merging